samples/outbox/store-and-forward/Version_5/Sender/app.config

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
well if v4 of the transport comes out, and still targets v6 of the core, i would assume we would want both a v3 and a v4 of this sample...?